### PR TITLE
Added `attribute_to_creator` parameter to `/documents/pretranslate` API endpoint.

### PR DESCRIPTION
**Pull request by judithnursalim**
Fixes #

https://lilt.productboard.com/feature-board/167360-feature-backlog-all/features/4904748/detail

## Engineering Notes

- Regenerated client libraries to add API support for attribute-to-creator pre-translation.

### CODE REVIEW

File: docs/FilesApi.md [hunk 1]
```diff
@@ -114,7 +114,7 @@ Name | Type | Description  | Notes
 [[Back to top]](#) [[Back to API list]](../README.md#documentation-for-api-endpoints) [[Back to Model list]](../README.md#documentation-for-models) [[Back to README]](../README.md)
 
 # **get_files**
-> File get_files(id=id)
+> InlineResponse200 get_files(id=id)
```
Comment (tgallant) [inline docs/FilesApi.md:117]: hmm i wonder why `File` got changed to `InlineResponse200` in this PR. nothing has changed in the spec for the files endpoint. did the openapi-generator version change or something? could you see if you can figure out why that changed?

Comment (judithnursalim) [reply docs/FilesApi.md:117]: Hmmm, good catch. 

I noticed this commit in `front` that modified `File` related swagger docs:

https://github.com/lilt/front/commit/aff555ccad67eb28ffb9ab82a0e5cd6b096a42d0#diff-d30f236ae744363691f88a5a21da0bc0

I didn't consciously upgrade openapi-generator unless the node upgrade that front requires did it. Also found this explanation:

https://github.com/swagger-api/swagger-editor/issues/1319#issuecomment-300530614

I don't have the understanding to connect these dots, any thoughts @tgallant?

Comment (tgallant) [reply docs/FilesApi.md:117]: Ah, good catch. I didn't know that had been updated. We may have to update the spec so it doesn't rename things to `InlineResponse200`. Let me take a look at that.

Comment (judithnursalim) [reply docs/FilesApi.md:117]: Thanks!

Comment (tgallant) [reply docs/FilesApi.md:117]: could you try generating it with the spec from this branch? i think adding names to the schemas should fix it https://github.com/lilt/front/tree/fix/adds-names-to-swagger-schema

Comment (judithnursalim) [reply docs/FilesApi.md:117]: Hmm, still `InlineResponse200` in `FilesApi.md` 🤔

Screen Shot 2020-08-04 at 3 58 52 PM

**Review by connorguy — APPROVED** [review]
LGTM

**Review by tgallant — APPROVED** [review]
I'd like to fix this `InlineResponse200` thing but we can worry about it separately

**Comment by judithnursalim** [comment]
> I'd like to fix this `InlineResponse200` thing but we can worry about it separately

Created issue https://github.com/lilt/lilt-python/issues/17 to keep track of this.

**Pull request merged**